.gitignore

```diff
@@ -8,3 +8,7 @@
 CMakeCache.txt
 src/SeExpr/parser
 src/SeExprEditor/generated
+.vs
+.vscode
+CMakeLists.txt.user
+CMakeSettings.json
```

Comment:
Sorry about this commit, I just noticed it ...
If you want I can redo the PR and remove this, but: `.vs` and `.vscode` are folders created by Visual Studio and VS Code, `CMakeLists.txt.user` is created by QtCreator, and `CMakeSettings.json` by Visual Studio, so it should be safe to ignore those.